libraries/app/test/app_test.js

```diff
@@ -79,6 +94,13 @@ suite(testing.suiteName(), function() {
       assert.equal(res.headers['content-type'], 'text/plain; charset=utf-8');
     });
 
+    test('/__version__', async function() {
+      const res = await request.get('http://localhost:1459/api/test/v1/__version__');
```

Comment:
https://github.com/mozilla-services/Dockerflow/#containerized-app-requirements specifies that the path be `/__version__`, not `/api/test/v1/__version__`.